src/Text/Pretty/Simple/Internal/Printer.hs

```diff
@@ -107,7 +107,7 @@ data OutputOptions = OutputOptions
   -- ^ If this is 'Nothing', then don't colorize the output.  If this is
   -- @'Just' colorOptions@, then use @colorOptions@ to colorize the output.
   --
-  , outputOptionsStringStyle :: StringOutputStyle
+  , outputOptionsPostProcess :: [Expr] -> [Expr]
```

Comment:
Should we capitalise the second `P`?

Reply:
I think `preprocess` and `postprocess` are used frequently now, which means the second `P` shouldn't be capitalized.